packages/appstore/src/components/wallet-account/wallet-account.tsx

```diff
@@ -0,0 +1,232 @@
+                                    app_icon='DTrader'
+                                    app_title='DTrader'
+                                />
+                                <div className='wallet-account__deriv-apps--seperator' />
+                                <PlatformLauncher
+                                    app_desc='Automate your trading, no coding needed.'
+                                    app_icon='DBot'
+                                    app_title='DBot'
+                                />
+                                <div className='wallet-account__deriv-apps--seperator' />
+                                <PlatformLauncher
+                                    app_desc='Our legacy options trading platform.'
+                                    app_icon='SmartTrader'
+                                    app_title='SmartTrader'
+                                />
+                                <div className='wallet-account__deriv-apps--seperator' />
+                                <PlatformLauncher
+                                    app_desc='Our legacy automated trading platform.'
+                                    app_icon='BinaryBot'
+                                    app_title='Binary Bot'
+                                />
+                                <div className='wallet-account__deriv-apps--seperator' />
+                                <PlatformLauncher
+                                    app_desc='Trade on the go with our mobile app.'
+                                    app_icon='DerivGo'
+                                    app_title='DerivGo'
+                                />
+                            </div>
+                        </div>
+                    </div>
+                </Accordion.Content>
+            </Accordion>
+
+            <Modal open={wallet_store.should_open_modal} onOpenChange={wallet_store.toggleModal}>
+                <Modal.Portal>
+                    <Modal.Overlay />
+                    <Modal.PageContent
+                        title='Trading account added'
+                        has_close_button={true}
+                        has_title_separator={true}
+                        has_footer_separator={false}
+                        should_prevent_close_on_click_outside={false}
+                        style={{ width: '60rem' }}
+                    >
+                        <div className='wallet-account__app-card-wrapper'>
+                            <AppstoreAppCard
+                                account_type='Demo'
+                                app_card_details={{
+                                    app_icon: 'icDxtradeSynthetic',
+                                    app_name: 'Deriv Apps',
+                                }}
+                                size='large'
+                                linked
+                            />
+                            <Text bold={false} type='paragraph-1' align='right'>
```

Comment:
If bold here always false, can we just not pass it?
```suggestion
                            <Text type='paragraph-1' align='right'>
```